src/model/__tests__/LeafList-test.ts

```diff
@@ -67,9 +67,10 @@ describe('Leaf List Model', () => {
     const leafList = new LeafList(testModel, {} as Container);
     const expectedType = {
       baseType: {
+        otherProps: new Map(),
         type: 'union',
         types: [
-          { type: 'uint32', range: { ranges: [{ min: 0, max: 999999999 }] } },
+          { type: 'uint32', range: { ranges: [{ min: 0, max: 999999999 }] }, otherProps: new Map() },
```

Comment:
Is it a convention to shorten properties to props or existing key name? I usually prefer spelling everything out

Reply:
`otherProps` was the convention used with leaf, leaf-list, etc. Decided it was best to keep the consistancy

Reply:
Yeah we use this same name for something similar elsewhere